src/vectors/tuple.lean

```diff
@@ -56,6 +56,9 @@ infix ` ** `:69 := tuple.scalar_mul
 
 def tuple.norm_sq {n : ℕ} (v : tuple n) : ℝ  := v ⬝ v
 
-def tuple.map: ∀ {n : ℕ}, (ℝ → ℝ) → tuple n → tuple n
```

Comment:
Why does this delete tuple.map?

Reply:
I have no clue why that happened. How do I fix that?